### PR TITLE
account for istiod scenario when troubleshooting mutatingwebhooks

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

File: content/en/docs/ops/common-problems/injection/index.md [hunk 1]
```diff
@@ -141,23 +141,37 @@ Warning  FailedCreate  3m (x17 over 8m)  replicaset-controller  Error creating:
 caused by an empty `caBundle` in the webhook configuration.
 
 Verify the `caBundle` in the `mutatingwebhookconfiguration` matches the
-   root certificate mounted in the `istio-sidecar-injector` pod.
+   root certificate mounted in the `istio-sidecar-injector` or `istiod`(if you're using istiod) pod.
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/injection/index.md:144]: Do we need to document this (and the other changes, below) two ways? Can't it just be switched to `istiod` now?
```suggestion
   root certificate mounted in the `iistiod` pod.
```

Comment (howardjohn) [reply content/en/docs/ops/common-problems/injection/index.md:144]: Istiod is the only option in 1.6

Comment (tariq1890) [reply content/en/docs/ops/common-problems/injection/index.md:144]: ok

**Comment by tariq1890** [comment]
What up with the netlify?

**Comment by tariq1890** [comment]
@frankbu Thanks for merging the PR. Why did netlify fail?

**Comment by frankbu** [comment]
@tariq1890 Don't know why, but netlify just seems to randomly fail sometimes, but currently the only way to rerun it is by pushing another commit, which is not very good if there's nothing else that needs to change in the PR. https://istio.slack.com/archives/C50V5EATT/p1588173408150300

**Pull request merged**